src/components/searchbox.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import elasticsearch from 'elasticsearch'
+
+let client = new elasticsearch.Client({
+	host: 'localhost:9200',
```

Comment:
We can change host accordingly when real time ? or should I add an alias and then change it in Client's Hosts file ?